src/StringWritableStream.js

```diff
@@ -56,7 +58,4 @@ class StringWritableStream {
   }
 }
 
-
-module.exports = {
-  StringWritableStream
-}
\ No newline at end of file
+module.exports = StringWritableStream;
```

Comment:
This is the main change?  It's a bit hard to see with all the quote changes.  Maybe a major version up if it changes the way you import it.

Reply:
This is the change. It changes the way its exported to correctly match whats actually been defined in `index.d.ts`. I can do a major change.